### ISSUE TITLE
init.el file issue

### ISSUE THREAD

**Issue by scrose5** [brittAnderson/Intro2Computing4Psychology#32]
Hello,

I am having a bit of trouble with the init.el file. I created the file and added the code provided on the emacs and python topics pages on Github (shown in the attached screenshot). When I try to run it using C-x C-e , it says nil. I tried to see if any of the packages needed updating but they didn't. I'm not sure what else could be causing this issue. If anyone has any suggestions, I would really appreciate it :)

![Screenshot_2020-10-24_22-02-06](https://user-images.githubusercontent.com/70968982/97094637-0efe4a00-1646-11eb-8170-450d246f2724.png)

**Comment by brittAnderson** [comment]
Make sure you do not have a file `~/.emacs`. If you do, delete it.
I think you may have an extra parentheses in there at the very end. Make sure they all match. Extra ones will screw things up. You can use `M-x check-parens` to help with that. 
Then when all is quit emacs and restart to make sure it re-reads the new version of your init.el file.

Let us know how it goes.

**Comment by scrose5** [comment]
I originally did search for the ~/.emacs.d to delete it and didn't find one, but I did this again to make sure this wasn't the issue, but again I didn't have another file like that. I also double-checked the parentheses and they all match up. I believe this could be an issue with the use-package command. When I opened my emacs I got an error message saying the function for this command was void (first screenshot). When I started emacs from the terminal with the "--debug-init" option, I got information shown in the second screenshot. 

I was originally thinking this could be the issue. This isn't going to sound very technical but in your video the "use-package" command appears in 
![Screenshot_2020-10-26_20-55-57](https://user-images.githubusercontent.com/70968982/97228371-ba3a0b00-17ce-11eb-8c0a-3b8f59fac96b.png)
![Screenshot_2020-10-26_20-56-51](https://user-images.githubusercontent.com/70968982/97228383-bf975580-17ce-11eb-9e30-b1ec3401baf6.png)

colour, whereas when I did it, it did not change colour. I ran into a similar issue when I did the original emacs assignment, but that was because I wasn't saving the file as an .sh file. In this case, I did name the file init.el, so I am a bit unsure why this isn't working.

**Comment by scrose5** [comment]
I originally did search for the ~/.emacs.d to delete it and didn't find one, but I did this again to make sure this wasn't the issue, but again I didn't have another file like that. I also double-checked the parentheses and they all match up. I believe this could be an issue with the use-package command. When I opened my emacs I got an error message saying the function for this command was void (first screenshot). When I started emacs from the terminal with the "--debug-init" option, I got information shown in the second screenshot.

I was originally thinking this could be the issue. This isn't going to sound very technical but in your video the "use-package" command appears in colour, whereas when I did it, it did not change colour. I ran into a similar issue when I did the original emacs assignment, but that was because I wasn't saving the file as an .sh file. In this case, I did name the file init.el, so I am a bit unsure why this isn't working.

![Screenshot_2020-10-26_20-55-57](https://user-images.githubusercontent.com/70968982/97228475-e48bc880-17ce-11eb-8803-1dc7a1972db5.png)
![Screenshot_2020-10-26_20-56-51](https://user-images.githubusercontent.com/70968982/97228486-ebb2d680-17ce-11eb-8c1d-430947d68acc.png)

**Comment by brittAnderson** [comment]
There is a difference between `~/.emacs.d` and `~/.emacs` Coding is a lot about details. I once spend two whole days many years ago trying to fix a matlab analysis routine that eventually came down to a single period. I had typed `*.` where I needed to have typed only `*`.  A few memories like that and you become a very careful reader. 

Make sure that you do not have a file `/home/xubuntu/.emacs`. Then make sure you DO have a _directory_ called /home/xubuntu/.emacs.d/` Then make sure that in that directory is the file `init.el`

As several of you are having trouble with this emacs start up stuff maybe we should have a video office hour. Why not get with your classmates virtually to see if any of you (but let's say less than 6) want to coordinate a time to meet virtually to go through this together to get it fixed.

**Comment by scrose5** [comment]
I checked for .emacs, and a file or directory doesn't exist. I do have a .emacs.d directory. I deleted the init.el file I originally created to try to start over, and made a new init.el file. I still receive the same error message from emacs when I open it after adding the code to the init.el file: "symbol's function definition is void: use-packages".

I think a virtual office hour may be beneficial. I have already met with a couple students and we were unable to figure it out amongst ourselves.
